src/Yasumi/Provider/Ukraine.php

```diff
@@ -83,24 +168,6 @@ private function calculateChristmasDay(): void
         ));
     }
 
-    /**
-     * International Workers' Day.
-     *
-     * @link https://en.wikipedia.org/wiki/International_Workers%27_Day#Ukraine
-     *
-     * @throws InvalidDateException
-     * @throws \InvalidArgumentException
-     * @throws UnknownLocaleException
-     * @throws \Exception
-     */
-    private function calculateSecondInternationalWorkersDay(): void
```

Comment:
If a holiday is not celebrated observed any longer, please don't delete it but provide a condition that it has been ended since a particular date/year.

Reply:
Reasonable!